fischkapp/src/components/styles/Header.styled.tsx

```diff
@@ -0,0 +1,64 @@
+import styled from "styled-components";
+
+export const StyledHeader = styled.div`
+  width: 100%;
+  height: 9vh;
+  position: fixed;
+  padding: 2.5vh;
+  box-shadow: 0px -16px 52px -10px rgba(152, 163, 207, 1);
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .header__logo {
+    display: flex;
+    height: 100%;
+    align-items: center;
+
+    .header__logoName {
+    }
+
+    .header__cardsNumber {
```

Comment:
**nitpick:** You don't need to use BEM naming when using styled-components. BEM is used for style scoping in big CSS files, but by using styled-components they are already scoped